src/templates/Accordion/Accordion.js

```diff
@@ -0,0 +1,62 @@
+import React, { createContext, useContext } from "react";
+import PropTypes from "prop-types";
+import classNames from "../../helpers/classNames";
+import useModal from "../../hooks/useModal";
+import { IconButton } from "../../components";
+import { Gear, Chevron } from "../../icons";
+import style from "./Accordion.scss";
+
+const AccordionContext = createContext({});
+
+const Accordion = ({ children }) => {
+  const [isShowing, toggle] = useModal();
+
+  return (
+    <AccordionContext.Provider value={{ isShowing, toggle }}>
+      {children}
+    </AccordionContext.Provider>
+  );
+};
+
+const AccordionTrigger = ({ children, className }) => {
+  const { isShowing, toggle } = useContext(AccordionContext);
+  const onClick = e => {
+    e.stopPropagation();
+    toggle();
+  };
+  return (
+    <div className={classNames(style.trigger, className)}>
```

Comment:
This is missing data-testid="accordion" prop. Test will fail from the accordion.test.js file